packages/code-du-travail-frontend/src/outils/common/StepList.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styled from "styled-components";
+import { FlatList, theme, Wrapper } from "@socialgouv/react-ui";
+
+export const STEP_LIST_WIDTH = "28rem";
+
+export function StepList({ activeIndex = 0, items = [] }) {
+  return (
+    <StyledWrapper variant="dark">
+      <Title>
+        Étape<Plural>s</Plural>
+        <StepProgress>
+          &nbsp;{`${activeIndex + 1}/${items.length}`}
+        </StepProgress>
+      </Title>
+      <FlatList>
+        {items.map((item, index) => (
+          <StyledListItem
+            key={item.name}
+            index={index}
+            isActive={activeIndex === index}
+          >
+            <IndexCircle isActive={activeIndex === index}>
+              {index + 1}
+            </IndexCircle>
+            {item.label}
+          </StyledListItem>
+        ))}
+      </FlatList>
+    </StyledWrapper>
+  );
+}
+
+const { spacings, breakpoints } = theme;
+
+const StyledWrapper = styled(Wrapper)`
+  position: absolute;
```

Comment:
why do you need position absolute here ? this looks more complicated than before

Reply:
It is slightly more complicated but I found no other ways to make it work responsively otherwise. The absolute + padding list is quite convenient though because it allows me to respect the full width background on figma even though the container has paddings. This is one is touchy, but if you take a look at the mobile / desktop version it should start to make sense

Reply:
I did, but I will do it, I probably miss something

Reply:
We can't get from a column display (mobile) to a mixed up layout (the way it is on desktop) with traditionnal flexboxes and grid is not ok on ie11

Reply:
https://photos.app.goo.gl/2KmB1bno2vG4MvNG6

Reply:
thanks for the clarification !